app/controllers/api/v1/jwt_api/people_controller.rb

```diff
@@ -23,8 +25,47 @@ def create
           render json: { status: 'Person created' }, status: :ok
         end
 
+        def list_children
+          render json: @children, each_serializer: Api::ChildSerializer
+        end
+
+        def update_child
+          # You cannot update the email address, because it does not send a new invitation.
+          @child = UpdateChildPerson.run(update_child_params.merge(person: @child, email: @child.email))
+          if @child.valid?
+            render json: @child.result, serializer: Api::ChildSerializer, status: :ok
+          else
+            render json: @child.errors, status: :bad_request
+          end
+        end
+
+        def destroy_child
+          if @child.account_active?
+            # Don't actually destroy the child's account, just make us no longer the parent.
```

Comment:
hmm this seems weird. Why do it like this? Wouldn't we want to make this more explicit? Perhaps because of gdpr reasons where we actually want to destroy someone?

Reply:
I was thinking more along the lines of that we would never want to allow for a person to destroy someone else's account, no matter the circumstances. 

I can ask the IKIA people about what they think about this.